### DISCUSSION TITLE
Do you have the demo website open-sourced somewhere?

### DISCUSSION THREAD

**Discussion by mnmkng** [DivanteLtd/vue-storefront#6724] [Q&A] [answered]
Hi, I need to quickly set up a running online store for demo purposes and I really like your demo site. Is the code available somewhere so that I could deploy it myself?

**Comment by mnmkng** [comment, marked as answer]
I guess I should've installed the boilerplate first. It's almost the same and should be sufficient. Sorry for typing too fast.

**Reply by filipsobol** [reply]
The [Boilerplate repository](https://github.com/vuestorefront/ecommerce-integration-boilerplate) is meant for creating custom integrations. If you want to generate a project using one of our [existing integrations](https://docs.vuestorefront.io/v2/integrations/), follow the [Installation guide](https://docs.vuestorefront.io/v2/getting-started/installation.html).
